geopandas/tests/test_plotting.py

```diff
@@ -108,13 +114,13 @@ def test_style_kwargs(self):
 
         # markersize
         ax = self.points.plot(markersize=10)
-        assert ax.collections[0].get_sizes() == [10]
+        assert ax.collections[0].get_sizes() == [100]
 
         ax = self.df.plot(markersize=10)
-        assert ax.collections[0].get_sizes() == [10]
+        assert ax.collections[0].get_sizes() == [100]
```

Comment:
If we don't change anything regarding the squaring the `markersize`, I'd suggest specifying here something like:
```
markersize = 10
# geopandas uses `mpl.pyplot.scatter` for plotting points, and so squares the markersize so 
# that markersizes are consistent with the sizing one would expect using `mpl.pyplot.plot` for points
scatter_size = markersize**2
```
and then use those variables in the subsequent tests.